exec/exploratory_plots.R

```diff
@@ -100,6 +100,12 @@ option_list <- list(
     metavar = "string",
     help = "A valid R palette name.",
     default = "Set1"
+  ),
+  make_option(
+    c("-l", "--log2_assays"),
+    type = "character",
+    default = NULL,
+    help = "Comma-separated list of assay_names to which to apply log2. Alternatively, comma-separated list of positive integers indicating which assays to log (1-based!). If not specified, the script will guess the log status based on the maximum value of the input data. If empty string, will not apply log2."
```

Comment:
R is always 1 based :-)

Reply:
I know that but maybe the user doesn't :D